data/remnant/remnant 2 side missions.txt

```diff
@@ -0,0 +1,149 @@
+			label paradisechoice
+			`	"Could I ask you something about life back in ancestral space?"`
+			choice
+				`	"Sure."`
+					goto askquestion
+				`	"Not right now."`
+			label vaillater
+			`	"Oh, maybe some other time, then." She looks disappointed, and settles in to eat."`
+				defer
+			label askquestion
+			`	"I have been reading through the history archive you brought back, and I came across a mention of Vail." She shakes her head in amusement. "The idea of an entire planet devoted to leisure activities seems so extravagant... Well, I am just really curious to see it. "Now, this is not a mission we are asking you to do, so I cannot offer any logistics compensation. I could, however, show you some of our history in exchange. Are you interested?"`
+			choice
+				`	"That sounds fun."`
+					goto vailfun
+				`	"I have done a bit of hoverboarding myself."`
+				`	"Maybe later."`
+					goto vaillater
+			apply
+				set "remnant: hoverboarding"
+			`	"Really? I noted a mention of that in the descriptions, and it looked challenging." Her commlink shifts to display a hologram of a hoverboarder executing a jump with an artistic poof of snow around them. "Anyway, I think this trip should be a good diversion for us both. I will meet you onboard the <ship> shortly."`
+				accept
+			label vailfun
+			`	"Excellent. I will meet you onboard the <ship> shortly."`
+				accept
+	on stopover
+		conversation
+			branch science
+				has "Research Laboratory"
+			`As you approach your navigation system easily picks up the beacons designating the approach path, and before too long you connect to the automated approach control that shunts you into the appropriate route down to a landing pad on the outskirts of the main spaceport.`
+				goto vaillanding
+			label science
+			`As you approach Vail your sensors light up with energy traces that it quickly identifies as part of the weather control network. From what you recall, many areas of the planet have been adjusted to have greater than usual snowfall. As you approach closer your navigation system easily picks up the beacons designating the approach path, and before too long you connect to the automated approach control that shunts you into the appropriate route down to a landing pad on the outskirts of the main spaceport.`
+			label vaillanding
+			`	Looking outside the ship, you can see a multitude of tourist-oriented facilities overwhelming the normal spaceport functions. You can pick out a small sign off to the side that indicates professional commodity and banking facilities, but otherwise the area is dominated by businesses offering mountain touring, sports classes, and vehicle rentals. Next to you, Taely alternates between peering out at the signs and reading brochures on a new commlink. "Here, <first>. This one sounds more interested in providing a challenge and less pampering. What do you think?"`
+			choice
+				`	"Looks good to me."`
+				`	"Alright, should be good."`
```

Comment:
"Rather than offer you the illusion of free choice, I will take the liberty of choo-sing for you."

Reply:
That is one thing I have learned from Mass Effect: Illusion of choice actually can be important: The vast majority of "choices" in Mass Effect have zero impact on the course of events of the game. They do, however, let the player shape their character. Am I the rebellious, crass, and aggressive commander that does stuff in the most gratingly hostile way I can? Am I the easy-going and/or professional soldier that accepts things the way they are and gets along with people? Some mix somewhere between the two? 

Exact same course of events, different story.

Reply:
This isn't the illusion of choice, though. You are offering the exact same choice twice. Like at this point, you might as well chuck out the choice node and just jump straight to the boarding.

Reply:
Fair, but me, as a player, really wants to be able to say something there.